packages/wrangler/src/__tests__/dev.test.tsx

```diff
@@ -369,22 +369,6 @@ describe("wrangler dev", () => {
 			expect(std.err).toMatchInlineSnapshot(`""`);
 		});
 
-		it("should fail for non-existing zones", async () => {
```

Comment:
@penalosa
I've removed this UT, since it does not seem to be correct when having Cloudflare for SaaS product in mind.
Please, let me know if it is ok to remove it.